backends/apple/coreml/scripts/install_requirements.sh

```diff
@@ -43,16 +47,7 @@ fi
 
 mkdir "$COREMLTOOLS_DIR_PATH/build"
 cmake -S "$COREMLTOOLS_DIR_PATH" -B "$COREMLTOOLS_DIR_PATH/build"
-cmake --build "$COREMLTOOLS_DIR_PATH/build" --parallel
-
-echo "${green}ExecuTorch: Installing coremltools."
-pip install "$COREMLTOOLS_DIR_PATH"
-
-STATUS=$?
-if [ $STATUS -ne 0 ]; then
-    echo "${red}ExecuTorch: Failed to install coremltools."
-    exit 1
-fi
+cmake --build "$COREMLTOOLS_DIR_PATH/build" --parallel --target mlmodel
```

Comment:
@YifanShenSZ @cymbalrush -- any reason why we still need to build mlmodel target from source, instead of using directly from pip install coremltools?

Reply:
@jathu 

Okay, let's not block it, but create an GH issue to track this...

Reply:
Done https://github.com/pytorch/executorch/issues/9651